java/bundles/org.eclipse.set.feature.plazmodel/src/org/eclipse/set/feature/plazmodel/export/ExportTopologischeCoordinatePart.java

```diff
@@ -0,0 +1,215 @@
+import org.eclipse.set.ppmodel.extensions.PunktObjektExtensions;
+import org.eclipse.set.ppmodel.extensions.container.MultiContainer_AttributeGroup;
+import org.eclipse.set.utils.BasePart;
+import org.eclipse.set.utils.table.export.ExportToCSV;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.locationtech.jts.geom.Coordinate;
+
+import com.google.common.collect.Streams;
+
+import jakarta.inject.Inject;
+
+/**
+ * Export topological coordinate of {@link Punkt_Objekt}, which calculate
+ * through {@link GeoCoordinateValid}
+ * 
+ * @author truong
+ */
+@SuppressWarnings("nls")
+public class ExportTopologischeCoordinatePart extends BasePart {
+	@Inject
+	IEclipseContext context;
+
+	@Inject
+	UserConfigurationService userConfigService;
+
+	@Inject
+	@Translation
+	Messages messages;
+
+	private static String HEADER_PATTERN = """
+			PlaZ Modell-Prüfung Topologische Coordinate
+			Datei: %s
+			Prüfungszeit: %s
+
+
+			"Lfd. Nr.";"Zustand";"Objektart";"Guid";"Koordinatensystem";"x";"y"
+			""";
+
+	ExportToCSV<String> csvExport;
+
+	/**
+	 * Default Constructor
+	 */
+	public ExportTopologischeCoordinatePart() {
+		super();
+		csvExport = new ExportToCSV<>(HEADER_PATTERN);
+	}
+
+	@Override
+	protected void createView(final Composite parent) {
+		final Button exportButton = new Button(parent, SWT.PUSH);
+		exportButton.setText(
+				"Exportieren Topologische Coordinate von Punkt_Objekt (csv)");
+		exportButton.addListener(SWT.Selection, event -> exportHandle());
+	}
+
+	private void exportHandle() {
+		final Optional<String> optionalOutputDir = getDialogService()
+				.selectDirectory(getToolboxShell(),
+						userConfigService.getLastExportPath().toString());
+
+		try {
+			getDialogService().showProgress(getToolboxShell(),
+					monitor -> optionalOutputDir.ifPresent(outputDir -> {
+						monitor.beginTask(
+								messages.PlazExport_ExportProcess_Message,
+								IProgressMonitor.UNKNOWN);
+						final Path exportPath = getModelSession()
+								.getToolboxPaths()
+								.getTableExportPath("tologische_coordinate",
+										Path.of(outputDir), null,
+										ExportPathExtension.TABLE_CSV_EXPORT_EXTENSION);
+						exportCoordinateToCSV(exportPath);
+					}));
+			optionalOutputDir.ifPresent(dir -> {
+				getDialogService().openDirectoryAfterExport(getToolboxShell(),
+						Path.of(dir));
+				userConfigService.setLastExportPath(Path.of(dir));
+			});
+		} catch (InvocationTargetException | InterruptedException e) {
+			Thread.currentThread().interrupt();
+			getDialogService().error(getToolboxShell(), e);
+		}
+	}
+
+	private void exportCoordinateToCSV(final Path path) {
+		if (Files.exists(path, LinkOption.NOFOLLOW_LINKS) && !getDialogService()
+				.confirmOverwrite(getToolboxShell(), path)) {
+			return;
+		}
+		final List<TopologicalCoordinate> topologischeCoordinaten = getTopologicalCoordiante();
```

Comment:
@TruongQuangSB sticking to english

```suggestion
		final List<TopologicalCoordinate> topologicalCoordinates = getTopologicalCoordiante();
```

And there a bunch of other locations...